src/Components/ComponentPreviewTransformModel.cs

```diff
@@ -44,12 +44,7 @@ public class ComponentPreviewTransformModel
         /// <summary>
         /// Gets or sets the create date of the component preview transform.
         /// </summary>
-        public DateTime CreateDate { get; set; }
-
-        /// <summary>
-        /// Gets or sets the identity of the component type of the component preview transform.
-        /// </summary>
-        public long ComponentTypeId { get; set; }
+        public DateTime CreatedDate { get; set; }
```

Comment:
DB column is CreateDate so make sure that gets if you are changing this

Reply:
I already updated the script to rename this column, to keep the naming consistent.